web/src/components/feed/PostComments/index.tsx

```diff
@@ -109,11 +109,12 @@ export function PostComment({
         (comment.privacy == PrivacyType.PRIVATE
           ? ' card-notification--comment-private'
           : '') +
-        (isReply ? ' card-notification--reply' : '')
+        (isReply ? ' card-notification--reply' : '') + 
+        (sessionUser && (sessionUser.id == comment.author.id) ? ' card-notification--reply' : '')
```

Comment:
what exactly does this do? if there was a bug on the comments what is exactly the issue number?

Reply:
I'm going to kill that line.